addons/tmf_loadouts/loadouts/chdkz.hpp

```diff
@@ -0,0 +1,510 @@
+	primaryWeapon[] = {"CUP_lmg_PKM"};
+	magazines[] = {
+		LIST_5("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Yellow_M")
+	};
+};
+
+class mmgac : r {
+	displayName = "MMG Ammo Carrier";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_4("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Yellow_M")
+	};
+};
+
+class mmgtl : aar {
+	displayName = "MMG Team Leader";
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_Com"};
+	backpack[] = {"CUP_B_CivPack_WDL"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"ItemRadio"};
+	backpackItems[] = {
+		LIST_3("CUP_100Rnd_TE4_LRT4_762x54_PK_Tracer_Yellow_M"),
+		"ACE_EntrenchingTool"
+	};
+};
+
+class hmgg : car {
+	displayName = "HMG Gunner";
+	backPack[] = {"CUP_B_DShkM_Gun_Bag"};
+};
+
+class hmgac : r {
+	displayName = "HMG Ammo Carrier";
+	backPack[] = {"CUP_B_DShkM_TripodHigh_Bag"};
+};
+
+class hmgtl : car {
+	displayName = "HMG Team Leader";
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_Com"};
+	backPack[] = {"CUP_B_DShkM_TripodLow_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"ItemRadio"};
+};
+
+class matg : car {
+	displayName = "MAT Gunner";
+	backpack[] = {"CUP_B_RPGPack_Khaki"};
+	secondaryWeapon[] = {"CUP_launch_RPG7V"};
+	secondaryAttachments[] = {"cup_optic_pgo7v2"};
+	magazines[] += {
+		LIST_2("CUP_PG7VL_M")
+	};
+};
+
+class matac : r {
+	displayName = "MAT Ammo Carrier";
+	backpack[] = {"CUP_B_HikingPack_Civ"};
+	backpackItems[] = {
+		LIST_2("CUP_PG7VL_M"),
+		LIST_3("CUP_OG7_M")
+	};
+};
+
+class mattl : car {
+	displayName = "MAT Team Leader";
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_Com"};
+	backpack[] = {"CUP_B_RPGPack_Khaki"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"ItemRadio"};
+	backpackItems[] = {
+		LIST_2("CUP_PG7VL_M")
+	};
+};
+
+class hatg : car {
+	displayName = "HAT Gunner";
+	backPack[] = {"CUP_B_Metis_Gun_Bag"};
+};
+
+class hatac : r {
+	displayName = "HAT Ammo Carrier";
+	backPack[] = {"CUP_B_Metis_Gun_Bag"};
+};
+
+class hattl : car {
+	displayName = "HAT Team Leader";
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_Com"};
+	backPack[] = {"CUP_B_Metis_Tripod_Bag"};
+	items[] += {"ACE_MapTools"};
+	linkedItems[] += {"ItemRadio"};
+};
+
+class mtrg : car {
+	displayName = "Mortar Gunner";
+	secondaryWeapon[] = {"ace_csw_staticMortarCarry"};
+	backPack[] = {"B_Carryall_khk"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrac : r {
+	displayName = "Mortar Ammo Carrier";
+	backPack[] = {"B_Carryall_khk"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+};
+
+class mtrtl : car {
+	displayName = "Mortar Team Leader";
+	vest[] = {"CUP_V_O_Ins_Carrier_Rig_Com"};
+	secondaryWeapon[] = {"ace_csw_carryMortarBaseplate"};
+	items[] = { MTR_GEAR };
+	backPack[] = {"B_Carryall_khk"};
+	backpackItems[] = {
+		LIST_4("ACE_1Rnd_82mm_Mo_HE"),
+		"ACE_1Rnd_82mm_Mo_Illum",
+		"ACE_1Rnd_82mm_Mo_Smoke"
+	};
+	linkedItems[] += {"ItemRadio"};
```

Comment:
No spotting binoculars intended? Probably won't get much use, but could come in handy in certain cirumstances.